### DISCUSSION TITLE
Yoga Übungen / Asanas suchen und finden und nutzen.

### DISCUSSION THREAD

**Discussion by Mersag** [neuefische/questions#64] [Web] [answered]
Bitte um HILFE :Ich benötige  108 Yoga Übungen als Foto, hat jemand eine Ahnung wo es eine Datenbank dafür gibt ? 
- Wie groß müssen die Bilder sein, die Pixel basiert sind? 
- Gibt es eine API für Yoga? 

Mein Ziel: Einzelne Bilder ( Yoga Asana / Übung) aus den 108 Yoga Übungen für meine App zu nutzen. 
Bereits versucht: 108 Yoga Position auf Google Suche eingegeben und erhalte Poster.

Ich weiß nicht, wie ich dieses lösen kann?

Danke im Vorraus, Meral

**Comment by PReichetanz** [comment, marked as answer]
Hallo Meral,

eine Google Suche mit "yoga api" hat mir eine (offenbar für ein Capstone project in den USA geschrieben API) [hier](https://github.com/rebeccaestes/yoga_api/blob/master/yoga_api.json) und [die dazugehörige deployte Webseite](https://suitcasecoder.github.io/Yoga-API_Capstone/).

Vielleicht hilft das als Startpunkt weiter?

**Reply by Mersag** [reply]
Hallo Paul,

die API  bzw. link hier ist nicht zu öffnen, es erscheint Error. Und die deployte Webseite wurde bereits im Slack von einem unserer Teilnehmer geteilt. Vielen Dank für deine Hilfe Paul :-)

**Reply by PReichetanz** [reply]
> 

Ich habe den kaputten Link überarbeitet, jetzt sollte er zum richtigen GitHub-Repo führen.
